src/main/scala/net/snowflake/spark/snowflake/io/StageReader.scala

```diff
@@ -122,9 +136,6 @@ private[io] object StageReader {
       System.currentTimeMillis()
     )
     SnowflakeTelemetry.send(conn.getTelemetry)
```

Comment:
Where is the telemetry Data?
SnowflakeTelemetry.send() function only flush the buffer and sends cache the metrics to Snowflake.
To add new telemetry message, you should call SnowflakeTelemetry.addLog()

Reply:
This is not a new telemetry. It's to send the EGRESS for Arrow format.